.travis.yml

```diff
@@ -55,17 +55,15 @@ matrix:
           packages:
             - *linux_deps
 
-# TODO: Android support
-
-#    - name: Android export template (release_debug, Clang)
-#      stage: build
-#      env: PLATFORM=android TOOLS=no TARGET=release_debug CACHE_NAME=${PLATFORM}-clang EXTRA_ARGS="warnings=extra werror=yes"
-#      os: linux
-#      compiler: clang
-#      addons:
-#        apt:
-#          packages:
-#            - openjdk-8-jdk
+    - name: Android export template (release_debug, Clang)
+      stage: build
+      env: PLATFORM=android TOOLS=no TARGET=release_debug CACHE_NAME=${PLATFORM}-clang EXTRA_ARGS="werror=yes"
```

Comment:
I had to remove `warnings=extra` as currently it raised warnings in the thirdparty `vk_mem_alloc.h` from the NDK.
It should be fixed to build in a separate env with `disable_warnings()` as done in `drivers/vulkan/SCsub` (cc @pouleyKetchoupp).

Reply:
On Android, we don't build Vulkan source like we do on other platforms, and the error occurs when `vk_mem_alloc.h` is included in our source.

I'm going to fix it by using `GCC diagnostic ignored` when needed.